lib/python/gunittest/invoker.py

```diff
@@ -231,10 +231,29 @@ def run_in_location(self, gisdbase, location, location_type,
                                    universal_location_value=GrassTestLoader.universal_tests_value,
                                    import_modules=False)
 
+        # TODO: check if it can be None here or breaks before
+        if test_list_out:
+            # overwrite old list
+            with open(test_list_out, 'w') as file:
+                file.write("# you can outcomment tests in the input list \n")
+            # append each module to list
+            for module in modules:
+                with open(test_list_out, 'a') as file:
+                    file.write(module.abs_file_path + "\n")
+
         self.reporter.start(results_dir)
+
         for module in modules:
-            self._run_test_module(module=module, results_dir=results_dir,
-                                  gisdbase=gisdbase, location=location)
+            # TODO: check if it can be None here or breaks before
+            if test_list_in:
```

Comment:
Doesn't this mean that the tests will run if and only if `test_list_in` has been provided?

Reply:
Good point, I changed it